src/Workspaces/CSharp/Portable/Simplification/Reducers/AbstractCSharpReducer.AbstractReductionRewriter.cs

```diff
@@ -59,14 +59,13 @@ public void Dispose()
             private static SyntaxNode GetParentNode(SyntaxNode node)
             {
                 if (node is ExpressionSyntax expression)
```

Comment:
nit: can we just make this a switch expression?

Reply:
yes, i can do that in my other branch :)